docs/README.md

```diff
@@ -214,3 +214,5 @@ information.
 	  Provides a database interface for the Bitcoin block chain
     * [btcutil](https://github.com/btcsuite/btcutil) - Provides Bitcoin-specific
 	  convenience functions and types
+    * [peer](https://github.com/btcsuite/btcd/tree/master/peer) -

```

Comment:
Would prefer this to come after the `wire` entry since it more logically fits there.